dist/index.js

```diff
@@ -7557,7 +7557,14 @@ function getRequiredRosDistributions() {
     return requiredRosDistributionsList;
 }
 //list of valid linux distributions
-const validDistro = ["noetic", "humble", "iron", "jazzy", "rolling"];
+const validDistro = [
+    "noetic",
+    "humble",
+    "iron",
```

Comment:
Should we remove iron here or better in a separate PR?

Reply:
I would do that in a separate PR.